src/tabcenter.css

```diff
@@ -281,6 +289,11 @@ img, .tab *:not(.clickable) {
     transition: background-position 150ms ease-out;
 }
 
+/*Adds a slight gap before the tab's text*/
+#tablist.shrinked .tab-context{
+  margin-right: 5px;
```

Comment:
I think this is a valuable change that belongs in its own commit though, drop the comment too.